### ISSUE TITLE
Marker ID

### ISSUE THREAD

**Issue by smiesnyrobert** [liodali/osm_flutter#498]
Hello,
can u please add some identificator for markers? id or something?

I need delete/update markers by id, not by lat long

I have many markers and is impossible maintain it without id

**Comment by smiesnyrobert** [comment]
hello do you plan to work on it in the near future? thanks

**Comment by liodali** [comment]
yeah,Im trying to stablize apis and i will start working on this very soon
